Files/ViewModels/CurrentInstanceViewModel.cs

```diff
@@ -11,11 +12,14 @@ public class CurrentInstanceViewModel : ObservableObject
          * values being manipulated inside the setter blocks.
          */
 
+        public IJsonSettings JsonSettings { get; }
```

Comment:
Looks like settings of json. Rename `JsonSettingsViewModel` to `BundlesSettings` and use instead of interface.

Reply:
By the way, this can be moved to the App.xaml.cs

Reply:
The interface `IJsonSettings` contains all availabe to modify settings that are saved to .json file. No, I will not rename `JsonSettingsViewModel` to `BundlesSettings` since I designed this class to be able to serialize more settings than just `SavedBundles`.

Reply:
I moved it to App.xaml.cs though

Reply:
@d2dyno1 Since we would like widgets to be a decoupled from the app as possible, it's best to make the change @tsvietOK requested.

Reply:
Should I name it `WidgetsSettingsViewModel`? @yaichenbaum

Reply:
@d2dyno1 I would name it `BundlesSettingsViewModel`. Other widgets moving forward will be extensions rather than built into Files so a common view model for them will not be necessary.

Reply:
I've decoupled it @yaichenbaum  @tsvietOK